src/Commands/Multidev/ListCommand.php

```diff
@@ -12,26 +12,26 @@ class ListCommand extends TerminusCommand implements SiteAwareInterface
     use SiteAwareTrait;
 
     /**
-     * List the multidev environments belonging to the given site
+     * Lists a site's Multidev environments.
      *
      * @authorize
      *
      * @command multidev:list
      * @aliases multidevs
      *
      * @field-labels
-     *   id: ID/Name
-     *   created: Created
-     *   domain: Domain
-     *   onserverdev: OnServer Dev?
-     *   locked: Locked?
-     *   initialized: Initialized?
+     *     id: Name
```

Comment:
This line is why the CI is failing. `tests/features/multidev-list.feature` needs to be updated to reflect this.